### DISCUSSION TITLE
Donating Funds to TanStack/query

### DISCUSSION THREAD

**Discussion by Lfada** [TanStack/query#6920] [General]
### Describe the bug

Hi Query maintainers. I'm Lucas, a core contributor to the Drips project (https://www.drips.network/). I help recipients of donations through their learning and onboarding process.

The WAGMI maintainers have been using https://github.com/TanStack/query. It's in fact one of their most critical dependencies!

WAGMI core contributors appreciate the work you all do and want to make sure that you are all well-funded to keep doing it. For that reason, they have decided to donate some funds to your project through [Drips](https://www.drips.network/).

Who can I talk to about it?

cc: @tannerlinsley @TkDodo @lachlancollins @boschni 

### Your minimal, reproducible example

n/a

### Steps to reproduce

n/a

### Expected behavior

n/a

### How often does this bug happen?

Only once

### Screenshots or Videos

n/a

### Platform

n/a

### Tanstack Query adapter

None

### TanStack Query version

n/a

### TypeScript version

n/a

**Comment by tannerlinsley** [comment]
Happy to discuss! Ping me on [our discord server](https://tlinz.com/discord) and we can chat more there.
